apps/src/aichat/views/modelCustomization/CompareModelsDialog.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {useState} from 'react';
+
+import AccessibleDialog from '@cdo/apps/templates/AccessibleDialog';
+import {Heading3} from '@cdo/apps/componentLibrary/typography';
+import Button from '@cdo/apps/componentLibrary/button/Button';
+
+import ModelDescriptionPanel from './ModelDescriptionPanel';
+import styles from './compare-models-dialog.module.scss';
+
+const CompareModelsDialog: React.FunctionComponent<{onClose: () => void}> = ({
+  onClose,
+}) => {
+  const [chosenModelLeft, setChosenModelLeft] = useState<string>('llama2');
+  const [chosenModelRight, setChosenModelRight] = useState<string>('mistral');
+
+  return (
+    <AccessibleDialog onClose={onClose} modalId={styles.modelComparisonDialog}>
```

Comment:
Getting an ID from a .module.scss file isn't a pattern I see elsewhere, which usually isn't a good sign 😛 .

I wanted a CSS selector that is more specific than the class names generated within AccessibleDialog's own styling, so ended up with this. It seemed like passing in another class name wouldn't guarantee that it would be more specific than what is applied by AccessibleDialog. Is there a "right" way to do this?

Reply:
Not sure if there's a problem with using ID. Though, one more thing you can try doing is to write a htmlElement.Classname style (eg. `div.modelComparisonDialog`) in module styles. This way the priority of the styles will be higher than just `.modelComparisonDialog`

Reply:
As an alternative, you could use the `styles` prop for `AccessibleDialog` and add the modal styling from `accessible-dialogue` to class `modal` in `compare-models-dialog`. Since that styling is duplicated, maybe make a mix-in?

Reply:
Denys's approach seemed simpler, so rolled with that. LMK if it looks ok!